### PR TITLE
Missing data path

### PR DESCRIPTION
**Pull request by SachinNinganure**
/hold

### CODE REVIEW

**Comment by SachinNinganure** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.14-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:35]: Replace the pre & post chains with `workflow: openshift-qe-installer-gcp` 
https://steps.ci.openshift.org/workflow/openshift-qe-installer-gcp 
and would look like this:

```
- as: control-plane-24nodes
  cron: 0 9 * * 3
  steps:
    cluster_profile: gcp-qe
    env:
      WORKER_REPLICA_COUNT: "24"
    test:
    - chain: openshift-qe-control-plane
    workflow: openshift-qe-installer-gcp
```

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "9"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:48]: Replace the pre & post chains with `workflow: openshift-qe-installer-gcp` 
https://steps.ci.openshift.org/workflow/openshift-qe-installer-gcp

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "5"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:64]: I do not think we need this job.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:77]: Replace the pre & post chains with `workflow: openshift-qe-installer-gcp` 
https://steps.ci.openshift.org/workflow/openshift-qe-installer-gcp

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:90]: Replace the pre & post chains with `workflow: openshift-qe-installer-gcp` 
https://steps.ci.openshift.org/workflow/openshift-qe-installer-gcp

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    env:
+      WORKER_REPLICA_COUNT: "5"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    workflow: openshift-qe-network-policy
+  timeout: 8h0m0s
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:105]: Not sure, if we need this job, please remove it.

**Review by krishvoor — COMMENTED** [review]
Few comments:
1. Kindly update the workflow
2. Add the `reporter_config`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 6 above)
Comment (SachinNinganure) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:105]: @qiliRedHat @liqcui   you think we require network-policy test part of this job?

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml:105]: @SachinNinganure I think you copied this from the gcp 412 and 413 nightly config files, right? https://github.com/openshift/release/blob/3581fcc6a29f5dbfa20160e98b2eabb708f53807/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml#L104  
I think Sharada added it, I suggest asking her about some background https://github.com/openshift/release/commit/e1ebe6f1ee6650b993fdf342967faedc09b424ef

Comment (SachinNinganure) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: okay

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 7]
```diff
@@ -3749,24 +3738,267 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: gcp
+    ci-operator.openshift.io/cloud-cluster-profile: gcp-qe
+    ci-operator.openshift.io/variant: gcp-4.14-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.14-nightly-x86-data-path-9nodes
```
[the remark this answers is not in the document]
Comment (SachinNinganure) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3769]: made the changes

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 8]
```diff
@@ -3749,24 +3738,267 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: gcp
+    ci-operator.openshift.io/cloud-cluster-profile: gcp-qe
+    ci-operator.openshift.io/variant: gcp-4.14-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.14-nightly-x86-node-density-heavy-24nodes
```
[the remark this answers is not in the document]
Comment (SachinNinganure) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3953]: made the changes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 4 above)
Comment (SachinNinganure) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: made the changes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 6 above)
Comment (SachinNinganure) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: @svetsa-rh   would you please let know if we should keep this test.

**Comment by SachinNinganure** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.14-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 3 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: @krishvoor Can you pls clarify which job is your comment referring to here?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 6 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: @SachinNinganure yes, let us keep it but keep it as `always_run: false` like you did here.
@krishvoor Any reason why you thought we should remove this job?

**Comment by svetsa-rh** [comment]
@SachinNinganure Can you please rebase your PR and try the test again? It should not fail with that error again this time.
@paigerube14 merged a fix today removing write results https://github.com/openshift/release/pull/46159

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml (hunk 6 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: @svetsa-rh We are not doing anything similar on Self-Managed AWS/ROSA.
Do we see a value in keeping this around only for GCP?

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: @krishvoor There was some issue with the test and Mohit suggested that we turn it off temporarily until it is fixed. This is the reason I suggested we can leave the test here and keep it as `always_run: false`. 
I have already pinged Mohit separately in getting this added to self-managed aws and Rosa.

Comment (SachinNinganure) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml]: I am resolving the comment as we are going to keep this. thanks

**Pull request merged**